packages/extension-ui/src/components/SplashHandler.tsx

```diff
@@ -51,11 +55,21 @@ function SplashHandler({ children, className }: SplashHandlerProps): React.React
       return false;
     };
 
-    const timeoutId = setTimeout(setIsSplashOn, 2000, updateWithErrorLog);
+    const endVideo = () => {
+      setIsSplashOn(updateWithErrorLog);
```

Comment:
Can we rename the other function, cause it was rather unclear that it does what it does:
```suggestion
      setIsSplashOn(setFalseWithErrorLog);
```